examples/calc/ops.ts

```diff
@@ -14,15 +14,11 @@ function div(a: number, b: number): number {
   return a / b;
 }
 
-export const ops_list: { [x: string]: (a: number, b: number) => number } = {
+export const ops_list: {[x: string]: (a: number, b: number) => number} = {
```

Comment:
Same comment here with the complex type.

Reply:
Same goes here, I'll define an `interface`